### ISSUE TITLE
pretrained model for v1.2 model

### ISSUE THREAD

**Issue by lyc1993** [SeanNaren/deepspeech.pytorch#388]
Hi Sean,

I am trying to use the pre-trained model you provide to train my data. 
I believe I am using the latest v1.2 model and the pre-trained model is v1.1.
Therefore, I get the error:
 size mismatch for rnns.0.rnn.weight_ih_l0: copying a param of torch.Size([2400, 1312]) from checkpoint, where the shape is torch.Size([2400, 672]) in current model.
Is that possible for you to release the pretrained model for v1.2 mode please? Or is there any other why to solve this problem?

Tnank you

**Comment by NaveAtVoca** [comment]
#366

**Comment by guyco87** [comment]
Isn't the MaskConv Sequential second convolution layer should also have padding?

`self.conv = MaskConv(nn.Sequential( nn.Conv2d(1, 32, kernel_size=(41, 11), stride=(2, 2), padding=(0, 10)), nn.BatchNorm2d(32), nn.Hardtanh(0, 20, inplace=True), nn.Conv2d(32, 32, kernel_size=(21, 11), stride=(2, 1),  padding=(0, 5)), nn.BatchNorm2d(32), nn.Hardtanh(0, 20, inplace=True) ))`

When I try to run it without the padding it complains that kernel size is bigger than input size.

Besides, did you manage to get any results using the pertained model? I keep getting empty transcriptions with it...
